sdk/anomalydetector/ai-anomaly-detector-rest/src/responses.ts

```diff
@@ -15,196 +15,183 @@ import {
   LastDetectionResultOutput,
 } from "./outputModels";
 
-/** This operation generates a model with an entire series, each point is detected with the same model. With this method, points before and after a certain point are used to determine whether it is an anomaly. The entire detection can give user an overall status of the time series. */
-export interface DetectUnivariateEntireSeries200Response extends HttpResponse {
+/** The request has succeeded. */
+export interface UnivariateDetectUnivariateEntireSeries200Response extends HttpResponse {
   status: "200";
   body: EntireDetectResponseOutput;
 }
 
-export interface DetectUnivariateEntireSeriesDefaultHeaders {
-  /** error code */
-  "x-ms-error-code"?: string;
```

Comment:
Why is the header name different?

Reply:
@lmazuel  @catalinaperalta  Seems it's a bug in cadl, we have sample header name in both uvad and mvad swagger, please help check.

Reply:
I found we both defined error headers with different code:
`x-ms-error`:
https://github.com/Azure/azure-rest-api-specs/blob/8804b10cf61267b81e06ebccbd1dd46677a54425/specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/cadl/univariate/models.cadl#L177-L181

`x-ms-error-code`:
https://github.com/Azure/azure-rest-api-specs/blob/8804b10cf61267b81e06ebccbd1dd46677a54425/specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/cadl/multivariate/models.cadl#L363-L370